src/plugin.h

```diff
@@ -20,50 +20,57 @@
 #include <string.h>
 #include "eth_plugin_interface.h"
 
-// All possible selectors of your plugin.
-// EDIT THIS: Enter your selectors here, in the format X(NAME, value)
-// A Xmacro below will create for you:
-//     - an enum named selector_t with every NAME
-//     - a map named SELECTORS associating each NAME with it's value
-#define SELECTORS_LIST(X)                    \
-    X(SWAP_EXACT_ETH_FOR_TOKENS, 0x7ff36ab5) \
-    X(BOILERPLATE_DUMMY_2, 0x13374242)
+#define PUBLIC_KEY_LENGTH   32
+#define LISK_ADDRESS_LENGTH 20
+
+#define SELECTORS_LIST(X)                \
+    X(CLAIM_REGULAR_ACCOUNT, 0xf6de242d) \
+    X(CLAIM_MULTI_SIGNATURE_ACCOUNT, 0x2f559f68)
```

Comment:
question: what are the second param here?

Reply:
It is method ID of the smart contract method.